src/EzPlatformCoreBundle/bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformCoreBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * Generates the configuration tree builder.
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder The tree builder
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder();
+
+        $rootNode = $treeBuilder->root(EzPlatformCoreExtension::EXTENSION_ALIAS);
```

Comment:
Keep in mind that In Symfony 4 configuration root name should be passed via constructor instead of using `Symfony\Component\Config\Definition\Builder\TreeBuilder::root` method, otherwise deprecation warning is emitted. 

More information:
* https://symfony.com/blog/new-in-symfony-4-2-important-deprecations#deprecated-tree-builders-without-root-nodes
* https://github.com/symfony/symfony/blob/master/UPGRADE-4.2.md#config

Reply:
> Keep in mind that In Symfony 4 configuration root name should be passed via constructor instead of using `Symfony\Component\Config\Definition\Builder\TreeBuilder::root` method, otherwise deprecation warning is emitted.
> 
> More information:
> 
> * https://symfony.com/blog/new-in-symfony-4-2-important-deprecations#deprecated-tree-builders-without-root-nodes
> * https://github.com/symfony/symfony/blob/master/UPGRADE-4.2.md#config

I'm aware, I had it like this when targeting master, but for Sf 3.4 forward compatibility is not possible, so I had to change it back. Just need to remember to update it on merge up :(